frontend/app/routes/project/show/edit.js

```diff
@@ -15,14 +15,14 @@ export default Route.extend({
   },
 
   actions: {
-    save: async function(changeset) {                  
+    save: async function(changeset) {
       await changeset.validate();
 
       if (!changeset.isValid) return;
-      
+
       try {
         const project = await changeset.save();
-
+        this.set('project.transportationAnalysis', await project.transportationAnalysis);
```

Comment:
This was necessary to pull in the updates to transportationAnalysis that are triggered by project update. Not sure if there's some cleaner, more ember-data way to do this, but this worked.